server/service/src/dashboard/invoice_count.rs

```diff
@@ -124,7 +124,8 @@ fn invoices_count(
             invoice_filter = invoice_filter.verified_datetime(datetime_filter)
         }
         InvoiceStatus::Cancelled => {
-             invoice_filter = invoice_filter. verified_datetime(datetime_filter)
+            // TO-DO: I don't think this is correct
+            invoice_filter = invoice_filter.verified_datetime(datetime_filter)
```

Comment:
Unrelated, but came across it -- surely we should be excluding "Cancelled" invoices from this count altogether?

Should we update this with #6500?

Reply:
Yes pls!

Reply:
I think its called with a single status at a time so don't think it will matter, but definitely worth following up on....